front/src/component/today/main/TodayCocktail.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const TodayCocktail = (props) => {
+  return <img src={props.image} alt="오늘의 칵테일" />;
```

Comment:
```suggestion
const TodayCocktail = ({ image }) => {
  return <img src={image} alt="오늘의 칵테일" />;
```

이렇게도 적을 수 있답니다.

자바스크립트의 [비구조화 할당](https://grahams.tistory.com/182)을 찾아보면 좋을 것 같아요😬